railties/lib/rails/engine.rb

```diff
@@ -217,7 +217,7 @@ module Rails
   #   end
   #
   # If the +MyEngine+ engine is marked as isolated, +FooController+ only has
-  # access to helpers from +MyEngine+, and <tt>url_helpers</tt> from
+  # access to helpers from +MyEngine+, and +url_helpers+ from
   # <tt>MyEngine::Engine.routes</tt>.
```

Comment:
```suggestion
  # +MyEngine::Engine.routes+.
```

Reply:
Probably didn't combine the fully-qualified namespace method syntax into a single expression, so there are likely more